x-pack/plugins/monitoring/server/routes/api/v1/elasticsearch_settings/check/internal_monitoring.ts

```diff
@@ -4,46 +4,97 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { CatIndicesParams } from 'elasticsearch';
+// @ts-ignore
+import { RequestHandlerContext } from 'kibana/server';
+import { getIndexPatterns } from '../../../../../lib/cluster/get_index_patterns';
+
 // @ts-ignore
 import { handleError } from '../../../../../lib/errors';
 import { RouteDependencies } from '../../../../../types';
 
-export function internalMonitoringCheckRoute(_server: unknown, npRoute: RouteDependencies) {
+const queryBody = {
+  size: 15,
+  query: {
+    bool: {
+      filter: [
+        {
+          range: {
+            timestamp: {
+              gte: 'now-15m',
```

Comment:
Why 15m?

Reply:
I was thinking we can get couple of latest monitoring document just in case their collection interval is set to something very big. Same with `size: 15`